applications/vanilla/controllers/class.categoriescontroller.php

```diff
@@ -455,6 +456,7 @@ public function all($Category = '', $displayAs = '') {
         $this->addModule('DiscussionFilterModule');
         $this->addModule('BookmarkedModule');
         $this->addModule($CategoryFollowToggleModule);
+        $this->addModule('TagModule');
```

Comment:
I'm not a fan of this module creeping in everywhere by default. Can we keep this part in the plugin?

I think the line is we want all the ability to *use* tagging in core, but turning on "users adding / using arbitrary tags" stays in the plugin.